src/state.rs

```diff
@@ -82,9 +82,14 @@ impl ProgressState {
         pct.max(0.0).min(1.0)
     }
 
-    /// Returns the position of the status bar as `(pos, len)` tuple.
-    pub fn position(&self) -> (u64, u64) {
-        (self.pos, self.len)
+    /// Returns the position of the status bar.
```

Comment:
If we're going to make the fields public we should just get rid of getter(s).

Reply:
Removed these methods.